tests/integ/test_local_mode.py

```diff
@@ -320,6 +151,32 @@ def test_mxnet_local_mode(sagemaker_local_session, mxnet_full_version):
             mx.delete_endpoint()
 
 
+@pytest.mark.local_mode
+def test_mxnet_distributed_local_mode(sagemaker_local_session, mxnet_full_version):
```

Comment:
This looks unrelated, but I'm assuming there was just a missing test for MXNet?

Reply:
the only distributed test was TF legacy mode, so I figured I'd add an equivalent one in after removing the TF one. There were already MXNet tests covering the other local mode cases covered by the deleted TF tests, so that's why this is the only test that got added.